x-pack/plugins/security_solution/public/detections/components/rules/step_rule_actions/translations.tsx

```diff
@@ -29,17 +33,27 @@ export const NO_ACTIONS_READ_PERMISSIONS = i18n.translate(
   }
 );
 
-export const RULE_SNOOZE_DESCRIPTION = i18n.translate(
-  'xpack.securitySolution.detectionEngine.createRule.stepRuleActions.snoozeDescription',
+const ruleSnoozeDocsLink = `${
```

Comment:
I'm not 100% sure the [link](https://www.elastic.co/guide/en/kibana/master/create-and-manage-rules.html#controlling-rules) is the right one though I wasn't able to find any rule snooze related docs.

@nastasha-solomon should I use another link here?

Reply:
Morning/afternoon! Scratch what I said yeseterday. You can actually use [these existing docs](https://www.elastic.co/guide/en/security/master/rules-ui-management.html#edit-rules-settings) for editing rule settings. The anchor for this section is `edit-rules-settings`.